constants/styles.js

```diff
@@ -0,0 +1,28 @@
+export const BASE_STYLES = {
+  NORMAL: "NORMAL",
+  BOLD: "BOLD",
+  ITALIC: "ITALIC",
+  H1: "H1",
+  H2: "H2",
+  H3: "H3",
+  H4: "H4",
+  H5: "H5",
+  CODE: "CODE",
+  LIST: "LIST",
+  HYPERLINK: "HYPERLINK",
+}
+
+export const TAG_STYLES = {
+  b: BASE_STYLES.BOLD,
+  strong: BASE_STYLES.BOLD,
+  i: BASE_STYLES.ITALIC,
+  em: BASE_STYLES.ITALIC,
+  a: BASE_STYLES.HYPERLINK,
+  h1: BASE_STYLES.H1,
+  h2: BASE_STYLES.H2,
+  h3: BASE_STYLES.H3,
+  h4: BASE_STYLES.H4,
+  h5: BASE_STYLES.H5,
+  code: BASE_STYLES.CODE,
+  li: BASE_STYLES.LIST,
+}
```

Comment:
9. Conclusion: One thing I have eventually been struggling with is exactly what is this parser for; and ideally in a real project I would have more of an idea of that, so that I could decide where to focus efforts. For example, do we know where users might copy their text from most of the time? Indeed some edge cases came from which place I have been pasting the code (I tried with google docs, medium, gmail). Or, what styles do we want to parse and what styles do we not want to support through pasting? For example, in medium when pasting text there, the size of the text isn't taken into account and everything is the same size, so that could be something we want to do here, or not.